tests/foreman/cli/test_remoteexecution.py

```diff
@@ -224,6 +224,63 @@ def test_positive_run_custom_job_template_by_ip(self, fixture_vmsetup, module_or
             )
             raise AssertionError(result)
 
+    @pytest.mark.destructive
+    @pytest.mark.parametrize(
+        'fixture_vmsetup',
+        [{'nick': 'rhel7'}],
+        ids=['rhel7'],
+        indirect=True,
+    )
+    def test_positive_use_alternate_directory(self, fixture_vmsetup, module_org):
+        """Use alternate working directory on client to execute rex jobs
+
+        :id: a0181f18-d3dc-4bd9-a2a6-430c2a49809e
+
+        :expectedresults: Verify the job was successfully ran against the host
+
+        :customerscenario: true
+
+        :parametrized: yes
+        """
+        client = fixture_vmsetup
+        testdir = gen_string('alpha')
+        result = client.run(f'mkdir /{testdir}')
+        assert result.status == 0
+        result = client.run(f'chcon --reference=/var /{testdir}')
+        assert result.status == 0
+        result = ssh.command(
+            f"sed -i r's/^:remote_working_dir:.*/:remote_working_dir: \\/{testdir}/' \
+            /etc/foreman-proxy/settings.d/remote_execution_ssh.yml",
+        )
+        assert result.return_code == 0
+        result = ssh.command('systemctl restart foreman-proxy')
+        assert result.return_code == 0
+
+        command = f'echo {gen_string("alpha")}'
+        invocation_command = make_job_invocation(
+            {
+                'job-template': 'Run Command - SSH Default',
+                'inputs': f'command={command}',
+                'search-query': f"name ~ {client.hostname}",
+            }
+        )
+
+        try:
+            assert invocation_command['success'] == '1'
+        except AssertionError:
+            result = 'host output: {}'.format(
```

Comment:
can you use f'string here

Reply:
I think it would improve readability here a bit defining a variable to use. It will end up being a bit awkward to try and fit the join inside the f-string otherwise.

```
job_output = ' '.join(jobInvocation.get_output({blah})
result = f'host output: {job_output}'
```